docker/jaas/webclient/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8u102-jdk
+
+ENV JWC_HTTP_PORT 80
+ENV JWC_JDBC_DRIVER com.mysql.jdbc.Driver
+ENV JWC_JDBC_URL jdbc:mysql://172.18.128.228:3306/jaggerdb
+ENV JWC_JDBC_USER root
+ENV JWC_JDBC_PASS root
+
+COPY jaas-webclient.jar /com/griddynamics/jaas/jaas-webclient.jar
```

Comment:
don't like the name. webclient currently has nothing to do with jaas. So the name is confusing. jagger-webclient is much more appropriate name